kctf-operator/README.md

```diff
@@ -0,0 +1,18 @@
+This operator was created using Operator-SDK 0.18.
+
+Tutorial on how to use Operator-SDK 0.18: https://v0-18-x.sdk.operatorframework.io/docs/golang/quickstart/
+
+The Custom Resource Definition is created inside the folder "deploy/crds" and it's generated from the file "pkg/apis/kctf/v1alpha1/challenge_types.go".
+
+The operator logic is all inside "pkg/controller/challenge/", where the main file is "challenge_controller.go".
+
+Before running the operator, you have to register the CRD in the server by creating it: "kubectl create -f deploy/crds/kctf.dev\_challenges\_crd.yaml".
```

Comment:
the kctf setup scripts will take care of this eventually and users don't have to do anything, right? Maybe add a note about this?

Reply:
yes, ok, i will add a note. I just added if any of you wanted to test it and for the future if someone wants to change something there.